run_dir/static/js/worksets.js

```diff
@@ -130,43 +133,43 @@ $(".tabbable").on("click", '[role="tab"]', function() {
       });
       $('.expand-all').on('click', function () {
-        var reqText = {'Expand All': ['Collapse All', 'visible', 'glyphicon-plus-sign', 'glyphicon-minus-sign'],
-                        'Collapse All': ['Expand All', 'collapse', 'glyphicon-minus-sign', 'glyphicon-plus-sign']};
-        $('.expand-all').find('.glyphicon').removeClass(reqText[$('.expand-all').text()][2]);
-        $('#samples_table').find('tr').find('.glyphicon').removeClass(reqText[$('.expand-all').text()][2]);
-        $('.expand-all').find('.glyphicon').addClass(reqText[$('.expand-all').text()][3]);
-        $('#samples_table').find('tr').find('.glyphicon').addClass(reqText[$('.expand-all').text()][3]);
+        var reqText = {'Expand All': ['Collapse All', 'visible', 'fa-plus-circle', 'fa-minus-circle'],
+                        'Collapse All': ['Expand All', 'collapse', 'fa-minus-circle', 'fa-plus-circle']};
+        $('.expand-all').find('.fa').removeClass(reqText[$('.expand-all').text()][2]);
+        $('#samples_table').find('tr').find('.fa').removeClass(reqText[$('.expand-all').text()][2]);
+        $('.expand-all').find('.fa').addClass(reqText[$('.expand-all').text()][3]);
+        $('#samples_table').find('tr').find('.fa').addClass(reqText[$('.expand-all').text()][3]);
         $('#samples_table').find('tr').find('table').css('visibility', reqText[$('.expand-all').text()][1]);
         $('.expand-all').contents().filter(function(){ return this.nodeType == 3; }).first().replaceWith(reqText[$('.expand-all').text()][0]);
       });
     });
   }
   if($(this).attr('href')=='#tab_closed_worksets'){
-    $("#closed_ws_table_body").html('<tr><td colspan="4" class="text-muted"><span class="glyphicon glyphicon-refresh glyphicon-spin"></span> <em>Loading..</em></td></tr>');
+    $("#closed_ws_table_body").html('<tr><td colspan="4" class="text-muted"><span class="fa fa-sync fa-spin"></span> <em>Loading..</em></td></tr>');
     return $.getJSON('/api/v1/closed_worksets', function(data) {
       $("#closed_ws_table_body").empty();
       $.each(data, function(key, value) {
         if(!($.isEmptyObject(value))){
           var tbl_row = $('<tr>');
           tbl_row.append($('<td>').html(value['date_run']));
-          tbl_row.append($('<td>').html('<a href="/workset/'+key+'">'+key+'</font>'+'</a>'));
+          tbl_row.append($('<td>').html('<a class="text-decoration-none" href="/workset/'+key+'">'+key+'</font>'+'</a>'));
           tbl_row.append($('<td>').html(function() {
             var t = '';
             $.each(value['projects'], function(project, projval) {
               if(!t.trim()){
-                t = t + '<a href="/project/'+project+'">'+projval['project_name']+' ('+project+')</a>'+' <span class="glyphicon glyphicon-folder-close" style="color:#337ab7"></span>';
+                t = t + '<a class="text-decoration-none" href="/project/'+project+'">'+projval['project_name']+' ('+project+')</a>'+' <span class="fa fa-folder" style="color:#337ab7"></span>';
```

Comment:
Also, I know I added the color for this :-) but now the folders don't match the links, maybe we could change the folder colors to that of text-primary?

Reply:
Yeah I think I agree with you. We could either just change it specificly for this one or we could modify the warning color in the bootstrap file we use. Do you think the warning color will always be too bright?

Reply:
Yes, I think you meant to put this next to my other comment though :-D 

As a bg-color it would look nice, but not for plain text with a white bg...

Reply:
I moved the comment, trying to hide my mistakes, hehe

Reply:
I think putting the span inside the `<a>` tag and removing the `style` should do the trick.

Reply:
I also moved my comment :-D

That would work!

Reply:
Updated on stage!